### ISSUE TITLE
FindHDF5 failing

### ISSUE THREAD

**Issue by mclarsen** [LLNL/conduit#448]
Updating conduit in a LLNL code causes errors. I suspect that the input to the strip is not defined.

HDF5 its looking for:
`/usr/tce/packages/hdf5/hdf5-serial-1.8.17-gcc4.9.3/`

Errors:
```
Make Error at cmake/thirdparty/SetupHDF5.cmake:171 (string):
  string sub-command STRIP requires two arguments.
Call Stack (most recent call first):
  cmake/Setup3rdParty.cmake:120 (include)
  CMakeLists.txt:114 (include)

CMake Error at cmake/thirdparty/SetupHDF5.cmake:181 (string):
  string sub-command STRIP requires two arguments.
Call Stack (most recent call first):
  cmake/Setup3rdParty.cmake:120 (include)
  CMakeLists.txt:114 (include)
```
https://github.com/LLNL/conduit/blob/faa992314b886cc1ec458d16f5a8b86bac3eb0dd/src/cmake/thirdparty/SetupHDF5.cmake#L181

**Comment by cyrush** [comment]
yes the flags we are hunting for are completely empty for that install. 
I'll add some logic to avoid an error

**Comment by cyrush** [comment]
#449 will resolve. I will merge pending CI checks

**Comment by cyrush** [comment]
resolved in #449
